app/components/request_list.tsx

```diff
@@ -22,6 +22,7 @@ export interface RequestObj {
   disableInterceptor:React.ChangeEvent<HTMLButtonElement>;
   updateInterceptorStatus:React.ChangeEvent<HTMLButtonElement>;
   isInterceptorOn:object;
+  fetchResponse:React.MouseEvent<HTMLButtonElement>
```

Comment:
Since `fetchResponse` is defined on `onClick` event, it is supposed to be` React.MouseEvent` i feel. Ref :https://reactjs.org/docs/events.html#mouse-events